sherpa/optmethods/optfcts.py

```diff
@@ -721,18 +721,19 @@ def stat_cb1( pars ):
 
     x, fval, nfev, info, fjac = _saoopt.cpp_lmdif( stat_cb1, m, x, ftol, xtol, gtol, maxfev, epsfcn, factor, verbose, xmin, xmax, fjac )
 
-    fjac = numpy.reshape(numpy.ravel(fjac, order='F'), (m, n), order='F')
+    if info > 0:
```

Comment:
@dtnguyen2 - so the change here is that we only calculate `covar` and update several variables (which make it into the return value later on) if `info` is positive (which presumably means that `cpp_lmdif` didn't fail?